_docs/publishing_api_documentation/pubapis_github_wikis.md

```diff
@@ -118,8 +118,7 @@ It's easiest to install Git by <a href="https://desktop.github.com">installing G
 
 2. Configure Git with Github authorization. This will allow you to push changes without entering your username and password each time. See the following topics to set this up:
  * [Set up Git](https://help.github.com/articles/set-up-git/) (Note that when you configure your username, use your Github username, which will be something like tomjohnson1492 instead of "Tom Johnson".)
- * [Generating a new SSH key](https://help.github.com/articles/generating-a-new-ssh-key/)
- * [Adding a new SSH key to the ssh-agent](https://help.github.com/articles/adding-a-new-ssh-key-to-the-ssh-agent/)
+ * [Generating a new SSH key and adding it to the ssh-agent](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/)
```

Comment:
Looks like github merged the two articles (generating-a-new-ssh-key, adding-a-new-ssh-key-to-the-ssh-agent) into one.